README-NEURAL.md

```diff
@@ -0,0 +1,106 @@
+The neural CRF parser is a high-performing constituency parser.
+
+
+
+##Preamble
+
+The neural CRF parser is described in:
+
+"Neural CRF Parsing" Greg Durrett and Dan Klein. ACL 2015.
+
+It is an extension of the span parser described in
+
+"Less Grammar, More Features" David Hall, Greg Durrett, and Dan Klein. ACL 2014.
+
+and is based on the Epic parsing framework. See https://github.com/dlwh/epic
+for more documentation about the span parser and the Epic framework.
+See http://www.eecs.berkeley.edu/~gdurrett/ for papers and BibTeX.
+
+Questions? Bugs? Email me at gdurrett@eecs.berkeley.edu
+
+
+
+##Setup
+
+You need three things to run the neural CRF parser:
+
+1) The compiled .jar; run ```sbt assembly``` to produce this
+
+2) A treebank: the Penn Treebank or one of the SPMRL treebanks
+
+3) Some sort of word vectors. These can either be in the .bin format
+of Mikolov et al. (2013) or the .txt format of Bansal et al. (ACL 2014).  For
+English, the best performance comes from using Bansal et al.'s vectors:
+
+http://ttic.uchicago.edu/~mbansal/codedata/dependencyEmbeddings-skipdep.zip
+
+For other languages, you can train suitable vectors on monolingual data using
+```word2vec``` with the following arguments:
+
+    -cbow 0 -size 100 -window 1 -sample 1e-4 -threads 8 -binary 0 -iter 15
+
+These are mildly tuned, and using a small window size is important, but other
+settings are likely to work well too.
+
+
+
+
+##Usage
+
+To run the parser on new text (tokenized, one-sentence-per-line), use the following command:
```

Comment:
ParseText actually tokenizes and sentences segments by default. To make your behavior, add the flags "--tokenizer whitespace --sentences newline"